### PR TITLE
Add support of VXLAN tunnel removal

### PR DESCRIPTION
**Pull request by pavel-shirshov**
**What I did**
Add support of vxlan removal.

**Why I did it**
To remove vxlan termination of WR restore

**How I verified it**
with test from sonic-mgmt.

**Details if related**

### CODE REVIEW

File: orchagent/vxlanorch.cpp [hunk 1]
```diff
@@ -658,15 +688,47 @@ bool VxlanTunnelOrch::addOperation(const Request& request)
...
 }
 
 bool VxlanTunnelOrch::delOperation(const Request& request)
 {
     SWSS_LOG_ENTER();
 
-    SWSS_LOG_ERROR("DEL operation is not implemented");
+    const auto& tunnel_name = request.getKeyString(0);
+
+    if(!isTunnelExists(tunnel_name))
```
Comment (prsunny) [inline orchagent/vxlanorch.cpp]: Spacing?

File: orchagent/vxlanorch.cpp [hunk 2]
```diff
@@ -658,15 +688,47 @@ bool VxlanTunnelOrch::addOperation(const Request& request)
...
+        SWSS_LOG_ERROR("Vxlan tunnel '%s' doesn't exist", tunnel_name.c_str());
+        return true;
+    }
+
+    auto tunnel_term_id = vxlan_tunnel_table_[tunnel_name].get()->getTunnelTermId();
+    try
+    {
+        remove_tunnel_termination(tunnel_term_id);
+    }
+    catch(const std::runtime_error& error)
```
Comment (prsunny) [inline orchagent/vxlanorch.cpp]: Spacing

File: orchagent/vxlanorch.cpp (hunk 1 above)
Comment (pavel-shirshov) [reply orchagent/vxlanorch.cpp]: I'd address it in separate PR

File: orchagent/vxlanorch.cpp (hunk 2 above)
Comment (pavel-shirshov) [reply orchagent/vxlanorch.cpp]: I'd address it in separate PR

**Pull request merged**